client/ayon_blender/plugins/publish/validate_no_animation.py

```diff
@@ -0,0 +1,45 @@
+
+import bpy
+
+from ayon_core.pipeline.publish import (
+    ValidateContentsOrder,
+    OptionalPyblishPluginMixin,
+    PublishValidationError
+)
+import ayon_blender.api.action
+from ayon_blender.api import plugin
+
+
+class ValidateNoAnimation(
+    plugin.BlenderInstancePlugin,
+    OptionalPyblishPluginMixin
+):
+    """Ensure that meshes don't have animation data."""
+
+    order = ValidateContentsOrder
+    hosts = ["blender"]
+    families = ["blendScene", "model", "rig"]
+    label = "No Animation"
+    actions = [ayon_blender.api.action.SelectInvalidAction]
+
+    @classmethod
+    def get_invalid(cls, instance):
+        invalid = []
+        for obj in instance:
+            if isinstance(obj, bpy.types.Object):
+                if obj.animation_data and obj.animation_data.action:
+                    invalid.append(obj)
+        return invalid
+
+    def process(self, instance):
+        if not self.is_active(instance.data):
+            return
+
+        invalid = self.get_invalid(instance)
+        if invalid:
+            names = ", ".join(obj.name for obj in invalid)
+            raise PublishValidationError(
+                "Objects found in instance which have"
+                f" animation data: {names}",
+                title="Keyframes on Objects"
```

Comment:
Can you implement a nice `description` for this validation error too? So supply a `description` argument with a nice markdown formatted wording what the validation is about, etc. similar to [this](https://github.com/ynput/ayon-core/blob/f2b67e3d79497b6f2c9b26fc33edd353a784235d/client/ayon_core/plugins/publish/validate_file_saved.py#L50-L60). That way the right side of Publisher UI will show a description about the validation itself.